scripts/build.sh

```diff
@@ -2,28 +2,15 @@
 
 set -euxo pipefail
 
-BUILD_TIME=$(date -u +%m-%d-%Y.%H-%M-%S.UTC)
-VERSION=$(cat version/CURRENT_VERSION | tr -d '\n')
-PKG_NAME=github.com/contiv/netplugin/version
-
-# BUILD_VERSION overrides the version from CURRENT_VERSION
-if [ -n "$BUILD_VERSION" ]; then
-	VERSION=$BUILD_VERSION
-fi
-
-if [ -z "$NIGHTLY_RELEASE" ]; then
-	BUILD_VERSION="$VERSION"
-else
-	BUILD_VERSION="$VERSION-$BUILD_TIME"
-fi
-
 GIT_COMMIT=$(./scripts/getGitCommit.sh)
 
-echo $BUILD_VERSION >$VERSION_FILE
+# TODO(chrisplo): remove when contiv/install no longer needs
+echo $BUILD_VERSION > netplugin-version
```

Comment:
What is this new filename for?  I don't see it referenced anywhere else

Reply:
VERSION was $NAME-version, name was netplugin

Reply:
let me double check, but I thought contiv/install uses it still